File: app/src/main/java/com/callmangement/ui/distributor/model/TranDate.java

package com.callmangement.ui.distributor.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class TranDate implements Serializable {

    @SerializedName("dayOfWeek")
    @Expose
    private String dayOfWeek;
    @SerializedName("dayOfYear")
    @Expose
    private Integer dayOfYear;
    @SerializedName("dayOfMonth")
    @Expose
    private Integer dayOfMonth;
    @SerializedName("hour")
    @Expose
    private Integer hour;
    @SerializedName("minute")
    @Expose
    private Integer minute;
    @SerializedName("month")
    @Expose
    private String month;
    @SerializedName("monthValue")
    @Expose
    private Integer monthValue;
    @SerializedName("nano")
    @Expose
    private Integer nano;
    @SerializedName("second")
    @Expose
    private Integer second;
    @SerializedName("year")
    @Expose
    private Integer year;
    @SerializedName("chronology")
    @Expose
    private Chronology chronology;

    public String getDayOfWeek() {
        return dayOfWeek;
    }

    public void setDayOfWeek(String dayOfWeek) {
        this.dayOfWeek = dayOfWeek;
    }

    public Integer getDayOfYear() {
        return dayOfYear;
    }

    public void setDayOfYear(Integer dayOfYear) {
        this.dayOfYear = dayOfYear;
    }

    public Integer getDayOfMonth() {
        return dayOfMonth;
    }

    public void setDayOfMonth(Integer dayOfMonth) {
        this.dayOfMonth = dayOfMonth;
    }

    public Integer getHour() {
        return hour;
    }

    public void setHour(Integer hour) {
        this.hour = hour;
    }

    public Integer getMinute() {
        return minute;
    }

    public void setMinute(Integer minute) {
        this.minute = minute;
    }

    public String getMonth() {
        return month;
    }

    public void setMonth(String month) {
        this.month = month;
    }

    public Integer getMonthValue() {
        return monthValue;
    }

    public void setMonthValue(Integer monthValue) {
        this.monthValue = monthValue;
    }

    public Integer getNano() {
        return nano;
    }

    public void setNano(Integer nano) {
        this.nano = nano;
    }

    public Integer getSecond() {
        return second;
    }

    public void setSecond(Integer second) {
        this.second = second;
    }

    public Integer getYear() {
        return year;
    }

    public void setYear(Integer year) {
        this.year = year;
    }

    public Chronology getChronology() {
        return chronology;
    }

    public void setChronology(Chronology chronology) {
        this.chronology = chronology;
    }

}

File: app/src/main/java/com/callmangement/ui/distributor/model/TranDate.kt
package com.callmangement.ui.distributor.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class TranDate : Serializable {
    @SerializedName("dayOfWeek")
    @Expose
    var dayOfWeek: String? = null

    @SerializedName("dayOfYear")
    @Expose
    var dayOfYear: Int? = null

    @SerializedName("dayOfMonth")
    @Expose
    var dayOfMonth: Int? = null

    @SerializedName("hour")
    @Expose
    var hour: Int? = null

    @SerializedName("minute")
    @Expose
    var minute: Int? = null

    @SerializedName("month")
    @Expose
    var month: String? = null

    @SerializedName("monthValue")
    @Expose
    var monthValue: Int? = null

    @SerializedName("nano")
    @Expose
    var nano: Int? = null

    @SerializedName("second")
    @Expose
    var second: Int? = null

    @SerializedName("year")
    @Expose
    var year: Int? = null

    @SerializedName("chronology")
    @Expose
    var chronology: Chronology? = null
}
